Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1660123347050)
+++ b/.idea/modules.xml	(date 1660123347050)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/Mastermind.iml" filepath="$PROJECT_DIR$/Mastermind.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1660123347050)
+++ b/.idea/.gitignore	(date 1660123347050)
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
Index: Mastermind.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Mastermind.iml b/Mastermind.iml
new file mode 100644
--- /dev/null	(date 1660123347050)
+++ b/Mastermind.iml	(date 1660123347050)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1660123347050)
+++ b/.idea/vcs.xml	(date 1660123347050)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/.." vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/Main.java
new file mode 100644
--- /dev/null	(date 1660283938166)
+++ b/src/Main.java	(date 1660283938166)
@@ -0,0 +1,53 @@
+import java.awt.*;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+    Scanner scanner1= new Scanner(System.in);
+    String Start= scanner1.nextLine();
+        System.out.println("Do you want to start the game?");           // Yes No if Schleife
+        String Yes = scanner1.nextLine();
+
+        Math.random();          // Braucht 6 Farben
+
+            System.out.println("The game has been started!");
+
+        String Nice= scanner1.nextLine();
+        System.out.println("Four random colors have been generated.");
+
+
+
+
+
+
+
+
+        Scanner color = new Scanner(System.in);
+
+        Color red = new Color(255, 0,0);            // 6 farben
+        Color blue = new Color(0, 0, 255);
+        Color green = new Color(0, 255,0);
+        Color gray = new Color(100,100,100);
+
+        switch(color) {             // if richtige Farbe+Ort
+            case "red":
+                System.out.println();
+                break;
+            case "blue":
+                System.out.println();
+                break;
+            case "green":
+                System.out.println();
+                break;
+            case "gray":
+                System.out.println();
+                break;
+
+        }
+
+
+
+
+    }
+}
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1660123347050)
+++ b/.idea/misc.xml	(date 1660123347050)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="corretto-17" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
